.gitignore

```diff
@@ -4,4 +4,3 @@ dist
 coverage
 .vscode
 .glee
-.next
```

Comment:
I think this is another accidental removal. Please add it back: https://github.com/asyncapi/glee/blob/master/.gitignore